node/resolvers/createReturnRequest.ts

```diff
@@ -69,12 +70,12 @@ export const createReturnRequest = async (
 
   const itemsToReturn = createItemsToReturn(items, orderItems)
 
-  // TODO: Apply the flag to return proportional shipping from admin
-  const shippingAmount = totals.find(({ id }) => id === 'Shipping')?.value ?? 0
+  const settings = await appSettings.get(SETTINGS_PATH, true)
```

Comment:
We can parallelize this call with the calls to get order and the request. If settings is empty, we should throw an error.